fpc_imp_yoshino_nile_tama.c

```diff
@@ -98,6 +98,7 @@ err_t send_modified_command_to_tz(fpc_data_t *ldata, struct qcom_km_ion_info_t i
     ion_fd_info.data[0].fd = ihandle.ifd_data_fd;
     ion_fd_info.data[0].cmd_buf_offset = 4;
 
+    *(uint32_t*)rec_cmd = 0;
```

Comment:
How about turning 
```c
void *rec_cmd = handle->ion_sbuffer + TZ_RESPONSE_OFFSET;
```
into
```c
int32_t *rec_cmd = (int32_t *)(handle->ion_sbuffer + TZ_RESPONSE_OFFSET);
```
And using that without cast here and below in `if (result = *rec_cmd)`?